src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
I do not have a strong option here:

- The old way is probably a bit more readable (but once you learn this idiom it's OK IMHO)
- If you cannot remember it you can still write the old `downcase` style and let `rubocop -a` fix it for you, the change is quick and cheap
- The original is a bit slower (but this is irrelevant in YaST)
- We should not diverge from the upstream Ruby style without any good reason

I'm not sure which item is the most important here...